test/functional/test_framework/mininode.py

```diff
@@ -654,3 +655,11 @@ def on_inv(self, message):
     def get_invs(self):
         with mininode_lock:
             return list(self.tx_invs_received.keys())
+
+    def wait_for_broadcast(self, txns):
+        # Waits for the txns (txids) to complete initial broadcast.
+        # Wait until invs have been received (and getdatas sent) for each txid.
+        self.wait_until(lambda: set(self.get_invs()) == set([int(tx, 16) for tx in txns]), 60)
+        # Flush messages and wait for the getdatas to be processed
+        # The mempool should mark unbroadcast=False for these transactions.
+        self.sync_with_ping()
```

Comment:
nit: pull out the description of the function from the implementation & description for readability 

```suggestion
    # Waits for the txns (txids) to complete initial broadcast.
    # The mempool should mark unbroadcast=False for these transactions.
    def wait_for_broadcast(self, txns):
        # Wait until invs have been received (and getdatas sent) for each txid.
        self.wait_until(lambda: set(self.get_invs()) == set([int(tx, 16) for tx in txns]), 60)
        # Flush messages and wait for the getdatas to be processed
        self.sync_with_ping()
```

Reply:
Good point, gonna follow [this PEP-8 guideline](https://www.python.org/dev/peps/pep-0008/#documentation-strings)